assets/training/model_management/src/azureml/model/mgmt/processors/transformers/config.py

```diff
@@ -103,3 +103,4 @@ class SupportedTasks(_CustomEnum):
     IMAGE_CLASSIFICATION = "image-classification"
     # ASR
     AUTOMATIC_SPEECH_RECOGNITION = "automatic-speech-recognition"
+    IMAGE_TEXT_TO_TEXT = "image-text-to-text"
```

Comment:
I also think, we don't need task name inside the transformers config. Can you please check.